docs/content/en/docs/getting-started/optional/autoscaling.md

```diff
@@ -35,10 +35,9 @@ Configure an EKS Anywhere worker node group to be picked up by a Cluster Autosca
           machineGroupRef:
             kind: VSphereMachineConfig
             name: worker-machine-b
-          count: 1
 ```
 
-Note that if no `count` is specified for the worker node group it will default to the `autoscalingConfiguration.minCount` value.
+Note that if `count` is specified for the worker node group, it's value will be ignored during cluster creation as well as cluster upgrade. If both `minCount` and `maxCount` are not specified, then autoscaling will not be configured for that worker node group and `count` will have a default value of `1`. If only one of `minCount` or `maxCount` is specified, then the other will have a default value of `0` and `count` will have a default value of `minCount`.
```

Comment:
If both aren't specified, do you mean if the `autoscalingConfiguration` field is specified with no fields underneath? The minCount and maxCount are pointers? And it's possible for minCount to be 0 if not specified and set the count to 0?

Reply:
If `autoscalingConfiguration` is specified as `null` in the cluster spec, then the asg struct will be a nil object and autoscaling won't be configured for that worker node group. If it's specified as an empty struct with curly brackets `{}`, then min count and max count will be defaulted to `0` and so count will also be defaulted to min count which is `0`.

Reply:
I will update the docs to remove that statement to make it less confusing.